src/main/resources/application.yml

```diff
@@ -7,16 +7,16 @@ springfox:
 
 spring:
   kafka:
-    bootstrap-servers: ${DATA_SYNC_KAFKA_BROKER_URL:localhost:9092}
+    bootstrap-servers: ${DATA_STREAM_KAFKA_BROKER_URL:localhost:9092}
```

Comment:
STREAMING_KAFKA_BROKER_URL maybe more appropriate name?

Reply:
addressed it